SimGeneral/MixingModule/python/stripDigitizer_cfi.py

```diff
@@ -23,6 +23,7 @@
     Noise = False, # will be added in DataMixer
     PreMixingMode = True, #Special mode to save all hit strips
     FedAlgorithm = 5, # special ZS mode: accept adc>0
+    includeAPVSimulation = False  # APV simulation is off for premix stage1
```

Comment:
is the comment correct? It says "premix stage1" but this is inside ```premix_stage2.toModify```
should the last line perhaps be  ```(premix_stage2 & ~tracker_apv_vfp30_2016).toModify(stripDigitizer, includeAPVSimulation = False)``` ?

Reply:
It should not read stage1 here. It's for the `MixingModule` instance used in the stage2 for signal digitization. See comment above from @makortel

Reply:
> It should not read stage1 here. It's for the `MixingModule` instance used in the stage2 for signal digitization. See comment above from @makortel

just to be sure (because in the reco meeting earlier today it appeared that this PR is ready to go as is), 
do I understand correctly that here the comment is wrong, but the value ```includeAPVSimulation = False``` for all kinds of premix_stage2 is correct.
Please clarify.

Reply:
Hi @slava77, yes that's correct. The only case in which `includeAPVSimulation = True` is needed is for the `PremixingModule` configuration which does the signal + PU overlay. See this comment: https://github.com/cms-sw/cmssw/pull/28355#discussion_r344835469. I updated the description in the latest commit.